backend/controller/controller.go

```diff
@@ -316,13 +328,20 @@ func (s *Service) ReplaceDeploy(ctx context.Context, c *connect.Request[ftlv1.Re
 	if err != nil {
 		return nil, connect.NewError(connect.CodeInvalidArgument, errors.WithStack(err))
 	}
+
```

Comment:
I think you'll want two loggers here? One for the old deployment, and one for the new one.